packages/session-ui/src/components/message-part.css

```
@@ -148,16 +148,19 @@
-     same pair in both schemes: an ink bubble on light, a near-white bubble on
-     dark. Padding is the site's px-4 py-2.5. */
+     of the ground — bg-deep/text-bg there. The app tokens
+     (--prompt-bubble-*, design-polish.css) realise that per scheme: the
+     site's ink bubble verbatim on light; on dark the inverse is seated at
+     ~85% luminance so a large light bubble doesn't glare on a dark field.
+     Fallback = raw inverse, for consumers without the app skin.
+     Padding is the site's px-4 py-2.5. */
   [data-slot="user-message-text"] {
     display: inline-block;
     white-space: pre-wrap;
     word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)